### ISSUE TITLE
Older and applied font mixes up

### ISSUE THREAD

**Issue by AnkurMal** [AmraniCh/github-code-font-changer#18]
The font messes up with the old font of github while selecting a text.

![Screenshot 2024-06-23 183111](https://github.com/AmraniCh/github-code-font-changer/assets/108927188/afa661c2-2abf-4f48-a47a-c3199698c0c2)

**Comment by AmraniCh** [comment]
Hey, thanks for opening this issue.

Can provide extra details about this issue ? what extension options you use ? if the file code in the screenshot is in public repository, can you send me the URL of that file please ?


I tested the same thing with dark mode enabled and the issue not happens with me! that weird!

**Comment by AnkurMal** [comment]
Font: JetBrains Mono
Weight: 400
Size: Not set to anything
Hide indentation guides is turned off
Browser is Chrome, OS is windows 11

[https://github.com/AnkurMal/Advent-of-Code-2023/blob/main/Day%204/part_one.rs](The link to the code)
(Although happens with any code)

Also, not using any extra extension for dark mode, its just Github's dark mode

**Comment by JiaChen-Zeng** [comment]
On my browser it's not mixed up but the selected text offsets a little bit

**Comment by AmraniCh** [comment]
@JiaChen-Zeng yes it should be just a little bit of text offset as a side effect of changing the original font

**Comment by AmraniCh** [comment]
@AnkurMal Can you confirm if you still have this issue please ? it been a while since this issue, and many updates has been made for Chrome and GitHub, otherwise we can close this issue

**Comment by AmraniCh** [comment]
Hey @AnkurMal, can you update the extension to the new version please and check if this issue remain ?
thank you
